src/MicrosoftTeams.ts

```diff
@@ -568,6 +561,17 @@ namespace microsoftTeams
                     {
                         // Open the window with a desired set of electron BrowserWindow features
                         childWindow = currentWindow.open(link.href, "_blank", "width=" + width + ", height=" + height);
+
+                        // Start monitoring the authentication window so that we can detect if it gets closed before the flow completes
+                        if (childWindow)
+                        {
+                            startAuthenticationWindowMonitor();
+                        }
+                        else
+                        {
+                            // If we failed to open the window fail the authentication flow
+                            handleFailure("FailedToOpenWindow");
+                        }
                     }
```

Comment:
Not: Besides the args to window.open, this is duped below. Might be worth considering extracting out to a separate function.